pygfx/__init__.py

```diff
@@ -18,6 +18,8 @@
 from .utils.viewport import Viewport
 from .utils import cm, logger
 
+from .utils.gallery_scraper import _get_sg_image_scraper
```

Comment:
I placed it here to make it more similar to a standard scraper and to allow downstream projects to use it. If you've built some application on top of pygfx you can set up your downstream `conf.py` using 

```python
sphinx_gallery_conf = {
    ...,
    "image_scrapers": ("pygfx",),
    "remove_config_comments": True,
}
```

and it will pick up our scraper and use it to build your gallery. More on this in the [sphinx-gallery docs](https://sphinx-gallery.github.io/stable/advanced.html#integrate-custom-scrapers-with-sphinx-gallery).

Reply:
Ok, that makes sense. Then my proposal would be to not import `gallery_scraper` by default, but implement 
`_get_sg_image_scraper` here that imports it when called. It looks like Sphinx-Gallery defined `_get_sg_image_scraper` to be used as a factory function to support this exact pattern.

Reply:
Sure, I'll move it to `__init__.py` :)